testsuites/syncgateway/functional/topology_specific_tests/multiple_servers/test_multiple_servers.py

```diff
@@ -22,7 +22,7 @@
 @pytest.mark.session
 @pytest.mark.channel
 @pytest.mark.bulkops
-@pytest.mark.skip(reason="Failing due to - https://github.com/couchbase/sync_gateway/issues/2173")
+# @pytest.mark.skip(reason="Failing due to - https://github.com/couchbase/sync_gateway/issues/2173")
```

Comment:
If the test is working, can the comment be removed?

Reply:
this test is working, so removed the comment